manuscript/deployChapter.md

```diff
@@ -1,39 +1,66 @@
 # Final Steps to Production
+# 距离上线部署仅有的最后一步
 
 The last chapters will show you how to deploy your application to production. You will use the free hosting service Heroku. On the way to deploy your application, you will learn more about *create-react-app*.
 
+在最后的章节中我们会向你展示如何部署你的应用到产品环境。你可以使用Heroku来免费托管和部署应用，在学习如何部署React应用的同时也可以了解更多*create-react-app*的相关特性。
+
 ## Eject
+## 弹出
 
 The following step and knowledge is **not necessary** to deploy your application to production. Still, I want to explain it to you. *create-react-app* comes with one feature to keep it extendable but also to prevent a vendor lock-in. A vendor lock-in usually happens when you buy into a technology but there is no escape hatch of using it in the future. Fortunately in *create-react-app* you have such an escape hatch with "eject".
 
+接下来的步骤和知识对于产品环境的部署来说*并不是必须*的。
```

Comment:
后面好像掉了几句话……

Reply:
vendor lock-in 和 escape hatch完全没法翻译，我就解释成“绑架”了

Reply:
Still, I want to explain it to you. *create-react-app* comes with one feature to keep it extendable but also to prevent a vendor lock-in. A vendor lock-in usually happens when you buy into a technology but there is no escape hatch of using it in the future. Fortunately in *create-react-app* you have such an escape hatch with "eject".
这一段的翻译不见了……

Reply:
不好意思刚刚该掉了，加回去了